dataloader/loader.py

```diff
@@ -146,7 +146,7 @@ def load_files_collection(
     doc._key = file["filename"]
     doc.set(file)
     doc["segmentnrs"] = segmentnrs
-    doc["parallelcount"] = parallelcount
+    doc["totallengthcount"] = totallengthcount
```

Comment:
`totallengthcount` does not exist in this scope. I get an error here when dataloading. Is this not supposed to be `doc["totallengthcount"] = parallelcount`?

Reply:
fixed.